code/script.js

```diff
@@ -1,13 +1,68 @@
+"use strict";
+
 // Variables that point to selected DOM elements
-const chat = document.getElementById('chat')
+const chat = document.getElementById('chat');
+const inputWrapper = document.getElementById('input-wrapper');
+const inputField = document.getElementById('name-input');
+const sendBtn = document.getElementById('send-btn');
+const formInputField = document.getElementById('name-form');
+
+// Variables where the user's choices will be stored.
+let teaBaseOption = "";
+let teaPearlOption = "";
+let drinkSizeOption = "";
+let totalOrderPrice = "";
+let customerName = "";
+let price = "";
 
-// If you need any global variables that you can use across different functions, declare them here:
+// Information about the different drink choices and prices
+const drinkOption = {
+  teaBase: ["Matcha", "Black Tea", "Oolong Tea"],
+  bobaPearlOption: ["Tapioca Pearls", "Brown Sugar Pearls", "Grass Jelly", "Coffee Jelly"],
+  drinkSize: ["Regular", "Large"],
+};
+
+const teaPrices = {  // in sek
+  matchaTeaPrice: [45],
+  blackTeaPrice: [40],
+  oolongTeaPrice: [43],
+}
 
+// FUNCTIONS
+
+// Function to clear the input field.
+const clearInputField = () => {
+  inputField.value = "";
+};
+
+/* Function to remove all child elements from a specified parent element. 
+"While" loop that continues as long as the inputWrapper element has child nodes. 
+The hasChildNodes() method checks if the inputWrapper element has any child nodes.
+inputWrapper.removeChild(inputWrapper.firstChild) inside the while loop removes the first child node/element of the parent element and does this iteratively until there are no more child nodes left.
+*/
+function removeChildrenFunction() {
+  while (inputWrapper.hasChildNodes()) {
+    inputWrapper.removeChild(inputWrapper.firstChild);
+  }
+};
 
-// Declare your functions after this comment
+// Function that causes the page to refresh.
+const reloadPage = () => {
+  setTimeout(() => {
+    location.reload();
+  }, 5000);
+};
 
-// This function will add a chat bubble in the correct place based on who the sender is
+// Function for bot sounds
+const playAudio = () => {
+  const notificationSound = document.getElementById('notificationSound');
+  notificationSound.volume = 0.7; // Set the volume to 70%
```

Comment:
super smart to set the volume with the notificationSound.volume, thanks for teaching me that!